lib/eslintTypescriptConfig.json

```diff
@@ -1,7 +1,7 @@
 {
   "parser": "@typescript-eslint/parser",
   "parserOptions": {
-    "project": ["./tsconfig.json"]
+    "project": true
```

Comment:
Ah yea that's for version 8. We're stuck on version 7 for now, so we have to use `project` which is the best option available for that version.

Reply:
Ah missed that sorry! Looking good tehn!

Reply:
No problem, I have a plan to get us on 8, but it's blocked on the eslint AirBNB packages updating their dependencies. It will coincide with ESLint v9 support too which is handy. I'm tracking it here: https://github.com/airbnb/javascript/issues/2961